playground/bicep/BicepSample.AppHost/Program.cs

```diff
@@ -15,12 +15,12 @@
 
 var kv = builder.AddAzureKeyVault("kv");
 var appConfig = builder.AddAzureAppConfiguration("appConfig").WithParameter("sku", "standard");
-var storage = builder.AddAzureBicepAzureStorage("storage");
+var storage = builder.AddAzureStorage("storage");
                     // .UseEmulator();
 
-var blobs = storage.AddBlob("blob");
-var tables = storage.AddTable("table");
-var queues = storage.AddQueue("queue");
+var blobs = storage.AddBlobs("blob");
```

Comment:
Worth keeping the old methods with obsolete tag?

Reply:
nah this one feels too easy to fix.